mutating-webhook/deployment/mutatingwebhook.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: admissionregistration.k8s.io/v1beta1
+kind: MutatingWebhookConfiguration
+metadata:
+  name: mutating-webhook-example-cfg
+  labels:
+    app: admission-webhook-example
+webhooks:
+  - name: mutating-example.banzaicloud.com
```

Comment:
We should change this to:

kubeplus.cloudark.io

Reply:
Actually, let go with following:
mutating-webhook.platform-as-code

Reply:
Sure

Reply:
Actually, lets change it to: crd-binding.kubeplus